libs/designer-ui/src/lib/tokenpicker/index.tsx

```diff
@@ -59,6 +59,15 @@ calloutStyles = isCopilotServiceEnabled()
     }
   : calloutStyles;
 
+const calloutStylesWithTopMargin: Partial<ICalloutContentStyles> = {
```

Comment:
This will conflict with changes in #5218. Can you make sure that the extended styles retain all styles from `calloutStyles` itself?